server/src/incrementalCompilation.ts

```diff
@@ -376,7 +376,7 @@ function triggerIncrementalCompilationOfFile(
       return;
     }
     const workspaceRootPath = projectRootPath
-      ? utils.findProjectRootOfFile(projectRootPath)
+      ? utils.findProjectRootOfFileInDir(projectRootPath)
```

Comment:
`findProjectRootOfFile` is used intentionally here, because it'll look up the project root in a way that doesn't touch the FS unless necessary. It's an optimization, because this is called frequently and it was causing pretty severe slowness at times.

So, the correct fix here would be to make `findProjectRootOfFile` work instead.

Reply:
This should be fixed now